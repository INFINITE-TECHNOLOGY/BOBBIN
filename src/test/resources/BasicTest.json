{
  "levels": "all",
  "classes": "all",
  "destinations": [
    {
      "name": "io.infinite.bobbin.destinations.FileDestination",
      "properties": {
        "fileKey": "threadName + level",
        "fileName": "\"./LOGS/${threadName}/${level}/${threadName}_${level}.log\"",
        "zipFileName": "\"./LOGS/${threadName}/${level}/ARCHIVE/${threadName}_${level}.zip\"",
        "cleanupZipFileName": "\"${origFileName}_${System.currentTimeMillis().toString()}.zip\""
      },
      "format": "\"${level}|${threadName}|${className}|${event.message}\\n\""
    },
    {
      "name": "io.infinite.bobbin.destinations.FileDestination",
      "properties": {
        "fileKey": "threadName + level",
        "fileName": "\"./LOGS/${threadName}/ALL_LEVELS/${threadName}.log\"",
        "zipFileName": "\"./LOGS/${threadName}/ALL_LEVELS/ARCHIVE/${threadName}.zip\"",
        "cleanupZipFileName": "\"${origFileName}_${System.currentTimeMillis().toString()}.zip\""
      },
      "format": "\"${level}|${threadName}|${className}|${event.message}\\n\""
    }
  ]
}