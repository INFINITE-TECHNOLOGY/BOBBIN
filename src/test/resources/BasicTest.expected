error|BasicTest|io.infinite.bobbin.tests_1_x_x.BasicTest|error abcd
warn|BasicTest|io.infinite.bobbin.tests_1_x_x.BasicTest|warn 1234
info|BasicTest|io.infinite.bobbin.tests_1_x_x.BasicTest|info abcd1234
debug|BasicTest|io.infinite.bobbin.tests_1_x_x.BasicTest|debug $uuid
trace|BasicTest|io.infinite.bobbin.tests_1_x_x.BasicTest|trace $uuid
